src/pages/office/__tests__/office.spec.ts

```diff
@@ -64,10 +64,13 @@ import {
 } from '../components/activity-code/activity-code.constants';
 import { ActivityCodes } from '../../../shared/models/activity-codes';
 import { CompleteTest, ValidationError } from '../office.actions';
-import { WelshTestChanged } from '../../../modules/tests/test-slot-attributes/test-slot-attributes.actions';
-import { LanguagePreferencesComponent } from '../components/language-preference/language-preferences';
 import { FormGroup, FormControl, Validators } from '@angular/forms';
 import { ToastControllerMock } from '../__mocks__/toast-controller-mock';
+import {
+  CandidateChoseToProceedWithTestInWelsh,
+  CandidateChoseToProceedWithTestInEnglish,
+} from '../../../modules/tests/communication-preferences/communication-preferences.actions';
+import { LanguagePreferencesComponent } from '../components/language-preference/language-preferences';
```

Comment:
This component isn't on the original cherry-pick - can you verify if this should be here?

Reply:
@rstepnowski The `LanguagePreferencesComponent`? - It has just moved location from above and was in release-2.0 before this cherry-pick